neo/app/assets/javascripts/index.js

```diff
@@ -90,15 +90,246 @@ app.controller('SearchCtrl', function($scope, $window, Asterix) {
   };
 });
 
-app.controller('MapCtrl', function($scope, $window, Asterix) {
+
+app.controller('MapCtrl', function($scope, $window, $http, $compile, Asterix, leafletData) {
```

Comment:
It seems that if we want to let angular perform on the html we insert using document.innerhtml, we need to compile it independently